examples/3D_lagrange_shbubcollapse/case.py

```
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python3
```

Ruff (0.14.5)

Shebang is present but file is not executable

(EXE001)